cmd/restart.go

```diff
@@ -0,0 +1,68 @@
+package cmd
+
+import (
+	"github.com/okteto/okteto/pkg/k8s/pods"
+	"github.com/okteto/okteto/pkg/log"
+
+	k8Client "github.com/okteto/okteto/pkg/k8s/client"
+	"github.com/okteto/okteto/pkg/model"
+
+	"github.com/spf13/cobra"
+)
+
+//Restart restarts the pods of a given dev mode deployment
+func Restart() *cobra.Command {
+	var namespace string
+	// get the name from it
+	var devPath string
+
+	cmd := &cobra.Command{
+		Use:   "restart COMMAND",
```

Comment:
Do we need to pass a command with restart? if not, it should just be the command name

```suggestion
		Use:   "restart",
```